libs/component-sample/src/lib/component-sample.module.ts

```diff
@@ -29,7 +31,8 @@ export const ROUTES: Routes = [
   { path: 'toolbar', component: ToolbarSampleComponent },
   { path: 'icon', component: IconSampleComponent },
   { path: 'sideNav', component: SideNavigationSampleComponent },
-  { path: 'searchResultList', component: SearchResultListSampleComponent }
+  { path: 'searchResultList', component: SearchResultListSampleComponent },
+  { path: 'formly', component: FormlyInputComponent },
```

Comment:
Why are you adding the test into sample component instead of its own test area?

Reply:
These are basic validations for all the types that's it added to the module.

Reply:
I don't think we need a separate sample area for that, that's why I added to it.

Reply:
Dave was supposed to talk with you about this. After a conversation we had.

Reply:
For this specific implementation formly will reside inside of the sideNav component, right? If so, then it makes sense to have it here. Once we start doing full form layouts, then formly samples could be a separate section completely.

Reply:
It will reside as a template inside not as an actually part of the control. Shilpa created four components; my point is that those components should have a demo page of functionality and usage. As a part of the process as a whole I can see it being apart of the layout sample.

Reply:
I agree we should create demo pages for sample forms. Does that need to be included in this pull request? I think he epics related to making formly work for the filters/sidenav. We could create another story to add examples for regular form pages in the next sprint.

Reply:
If one of them is being added here what about the other three.